spec/models/bread_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.describe Bread, type: :model do
   describe "relationships" do
     it { should belong_to(:country)}
+    it { should have_many(:user_breads) }
```

Comment:
Nice, I was wanting to get the PR up and spaced adding in what I added in the models.